expr/bitwise.go

```diff
@@ -54,6 +54,9 @@ func (e *Bitwise) marshal(fam byte) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	if dataOnly {
```

Comment:
Instead of adding a boolean to all marshal functions, could we split this logic out into two separate functions? One to prepare the data (and return it), another one which calls the data preparation func and then the netlink.MarshalAttributes?

Reply:
Agreed, refactored to separate functions approach.